cmd/sqlflowserver/main_test.go

```diff
@@ -1691,6 +1691,37 @@ INTO e2etest_keras_dnn_model_distributed;`, caseTrainTable, caseTestTable)
 	a.NoError(err)
 }
 
+func CasePassSelectedColsToPredictResultTable(t *testing.T) {
```

Comment:
Rename it like `CasePAIMaxComputeXXX`

Reply:
This case is inside `TestEnd2EndMaxComputePAI `, maybe we don't need the duplicate prefix?

Reply:
To be the same format as other cases, better to use the same name format.